cli/tests/conftest.py

```
@@ -0,0 +1,368 @@
+import pytest
+import os
+import tempfile
+import pandas as pd
+from unittest.mock import MagicMock, patch
+from pathlib import Path
```

Ruff (0.11.9)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)